### ISSUE TITLE
./gradlew assembleRelease throws dialogCornerRadius not found

### ISSUE THREAD

**Issue by Tbaut** [paritytech/parity-signer#327]
This error prevents us from building the apk with CI. I am still able to build it using Android Studio (magically).

Full error:
```bash
Execution failed for task ':react-native-secure-storage:verifyReleaseResources'.
> 1 exception was raised by workers:
  com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  /home/thib/github/parity-signer/node_modules/react-native-secure-storage/android/build/intermediate>s/res/merged/release/values-v28/values-v28.xml:7: error: resource android:attr/dialogCornerRadius not found.
  /home/thib/github/parity-signer/node_modules/react-native-secure-storage/android/build/intermediates/res/merged/release/values-v28/values-v28.xml:11: error: resource android:attr/dialogCornerRadius not found.
  /home/thib/github/parity-signer/node_modules/react-native-secure-storage/android/build/intermediates/res/merged/release/values/values.xml:2737: error: resource android:attr/fontVariationSettings not found.
  /home/thib/github/parity-signer/node_modules/react-native-secure-storage/android/build/intermediates/res/merged/release/values/values.xml:2738: error: resource android:attr/ttcIndex not found.
  error: failed linking references.
```

This is linked to our implementation of [react-native-secure-storage](https://github.com/paritytech/react-native-secure-storage). It could be related to the `+` [here](https://github.com/paritytech/react-native-secure-storage/blob/master/android/build.gradle#L33) as suggested 
https://stackoverflow.com/questions/49280632/error9-5-error-resource-androidattr-dialogcornerradius-not-found

**Comment by hanwencheng** [comment]
I am currently working on to use `react-native-keychain` to replace the `react-native-secure-storage`. I will also check this issue by the way. @Tbaut do you use the [this way](https://stackoverflow.com/a/36961021/3060739) to build the release?

I met error like following when I first try `react-native bundle`, did I miss any steps?
<img width="564" alt="Screenshot 2019-08-26 at 22 01 21" src="https://user-images.githubusercontent.com/6014309/63719428-30e71b80-c84d-11e9-9b39-ce6591c2408a.png">

**Comment by Tbaut** [comment]
I do 
```bash
$ cd android
$ ./gradlew assembleRelease
```

and this used to work fine, but since the update to v0.60 it throws the error in OP. No doubt that using an updated package in place of react-native-secure-storage, things would work better.

**Comment by hanwencheng** [comment]
As dissucssed, this should be fixed by upgrading react native > `0.61` .

**Comment by Tbaut** [comment]
This is solved :tada:
